build.sbt

```diff
@@ -73,6 +73,8 @@ scalacOptions ++= List(
 enablePlugins(ScriptedPlugin)
 sbtPlugin := true
 scriptedBufferLog := false
+scriptedBatchExecution := true
```

Comment:
despite the intent of https://github.com/sbt/sbt/pull/6151, `scriptedBatchExecution` is never true by default (because [`binVersion`](https://github.com/sbt/sbt/pull/6151/files#diff-6b2c28f765181fcabca3279c1f686a4ca39ece8ad09b2ec4281dc9287dc4fdc5R84) is "1" on sbt 1.x)